### ISSUE TITLE
revise high level description

### ISSUE THREAD

**Issue by jjallaire** [rstudio/tfestimators#87]
In the high level description we mention a couple of things which we may want to remove for now:

1) Experiment API (see https://github.com/rstudio/tfestimators/issues/86)

2) Ability to deploy models without an R runtime

The second is of course possible but we provide no documentation for it. Perhaps we shouldn't advertise this capability until we've properly tooled and documented it.

**Comment by terrytangyuan** [comment]
Agree that we can remove the first one. Maybe rephrase the second one to be something like "coming soon"?

**Comment by jjallaire** [comment]
I'd just remove the second one entirely we if we can't deliver anything
more concrete right now.

On Thu, Aug 31, 2017 at 9:38 AM, Yuan (Terry) Tang <notifications@github.com

**Comment by kevinushey** [comment]
https://github.com/rstudio/tfestimators/commit/0e0357c991243e51dbce88864adde5412a51c3ff
